docs/docs/cmd/planner/task/task-set.md

```diff
@@ -103,6 +103,115 @@ Marks a Microsoft Planner task with the ID _Z-RLQGfppU6H3663DBzfs5gAMD3o_ as 50%
 m365 planner task set --id "2Vf8JHgsBUiIf-nuvBtv-ZgAAYw2"  --percentComplete 50 --appliedCategories "category1,category3"
 ```
 
+## Response
+
+=== "JSON"
+
+    ```json
+    {
+      "planId": "oUHpnKBFekqfGE_PS6GGUZcAFY7b",
+      "bucketId": "vncYUXCRBke28qMLB-d4xJcACtNz",
+      "title": "Important task",
+      "orderHint": "8585269235419217847",
+      "assigneePriority": "",
+      "percentComplete": 50,
+      "startDateTime": "2023-01-20T00:00:00Z",
+      "createdDateTime": "2023-01-25T21:49:03.555796Z",
+      "dueDateTime": "2023-02-15T00:00:00Z",
+      "hasDescription": true,
+      "previewType": "noPreview",
+      "completedDateTime": null,
+      "completedBy": null,
+      "referenceCount": 0,
+      "checklistItemCount": 1,
+      "activeChecklistItemCount": 1,
+      "conversationThreadId": null,
+      "priority": 3,
+      "id": "OopX1ANphEu7Lm4-0tVtl5cAFRGQ",
+      "createdBy": {
+        "user": {
+          "displayName": null,
+          "id": "b2091e18-7882-4efe-b7d1-90703f5a5c65"
+        },
+        "application": {
+          "displayName": null,
+          "id": "31359c7f-bd7e-475c-86db-fdb8c937548e"
+        }
+      },
+      "appliedCategories": {},
+      "assignments": {}
+    }
+    ```
```

Comment:
This command returns a different response when they use the option `description`. Let's include both outputs similarly like `planner plan add`

[The docs article](https://pnp.github.io/cli-microsoft365/cmd/planner/plan/plan-add/)
[The MarkDown file location](https://github.com/pnp/cli-microsoft365/blob/main/docs/docs/cmd/planner/plan/plan-add.md?plain=1)